fedimint-cli/Cargo.toml

```diff
@@ -62,5 +62,5 @@ fedimint-build = { workspace = true }
 [lints]
 workspace = true
 
-[target.'cfg(not(any(target_env = "msvc", target_os = "ios")))'.dependencies]
+[target.'cfg(not(any(target_env = "msvc", target_os = "ios", target_os = "android")))'.dependencies]
```

Comment:
Should we just only turn it on on linux maybe? Or have a feature for it that's default on but can be turned off externally?

Reply:
Probably good idea, but up to @maan2003 if he wants to amend. If not, this PR is still makess it better than before.

Reply:
will do it follow up, and we don't need to backport that one

Reply:
https://github.com/fedimint/fedimint/pull/7422